action/organisation/user/create.go

```diff
@@ -60,7 +62,7 @@ func create(w http.ResponseWriter, r *http.Request) {
 
 	validationError := validationx.Check(req)
 	if validationError != nil {
-		util.Log.Error(validationError)
+		loggerx.Error(errors.New("validation error"))
```

Comment:
log validationError and not "validation error"

Reply:
validationError is array of Message and loggerx.Error() accepts error in parameter.